examples/js/fetch-ohlcv.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const ccxt = require ('../../ccxt.js');
```

Comment:
usually end users don't import ccxt like this

Reply:
Thanks! 
A) I can't say decisively, whether we need async & sync examples. I just come out from the point, that some users either use only sync version, thus 'async-only' examples wouldn't be help for them imo. if we just make `sync` only examples, then ppl which only use async version, will need async-example. So, in my current view, both versions are needed.
B) ofc, it's not needed to re-transpile it on every build. I didn't have implemented that, but ofc likewise we do derived exchange classes, we can cache/rebuild it only whenever there is change. But I dont think that caching mechanism would be in this same PR's scope, because :
- for local builds - yes, we can just make it like we do for derived exchange files (only transpiling the changed ones. I'll add commit for that).
- for travis builds - it might be slightly trickier than being said , bcz we will need somehow to make travis aware of previous states, which will need some local cache or etc.. So, maybe for this current moment, we skip caching issue and we can address that later?

Reply:
ah, I've just copied that line from another example-js.

Reply:
btw, how you suggest this to be? so the example files were runnable directly.